packages/hashes.csv

```diff
@@ -1,86 +1,43 @@
-fetchai/skills/generic_buyer/skill.yaml,QmTNALpDXH6QqxQGPCRoyaUZXP5AmAwQDJF9PeY2mG4EjQ
-fetchai/skills/generic_seller,QmWFxkLWoFz5exkyDAPh5SNCRQkLww8jGNCvEsRAadBXjg
-fetchai/skills/generic_seller/skill.yaml,QmeVAEhGEV1q8G5n3xnzcBZMR68wdJ8TD26TXEcTMA1qde
-fetchai/skills/gym,QmdW34AC41urZzDyFAt7BWByLibo5rXKH95JJntuhZpZyi
-fetchai/skills/gym/skill.yaml,QmZRARYMBZezvSdfACTo457KdcNANQ4DeMd6k4hretB5VX
-fetchai/skills/http_echo,QmbnmNpiQZLx2Q5s3agdscewYBUjqDQUjSZaFCPP3e1Nd6
-fetchai/skills/http_echo/skill.yaml,QmQ9vYQwoVXoHRt9CJAJwCvryS6oivcpvd5ongwCrvsH7z
-fetchai/skills/task_test_skill,QmYFTfauVZicfZum8vkDNYQCWdy8yvDMMqVtfNtXg741cu
-fetchai/skills/task_test_skill/skill.yaml,QmeEcvgSpMa921aVGZrjeKVNBnrj5SfDHGteMbkkgLEFj3
-open_aea/agents/gym_aea,QmWFbikjuc7QCN8gtT8Ug7i5B5Y8XtpuRSnQmqnZ882Yjk
-open_aea/agents/gym_aea/aea-config.yaml,QmXhJx9WLKxJ9YtFTP2NPinPUSeDxu1m7wpvhoKnLPPLDv
-open_aea/agents/http_echo,QmdS5D9mjEoBsrVkBoTFJhir8qB3Mp8SSqzVQbyBcQJVTe
-open_aea/agents/http_echo/aea-config.yaml,QmWTgEMBrnRED8NUxSg5YdckAu51SmhSbf1FqSe8ZwASdV
-open_aea/agents/my_first_aea,QmfNLofcJuxzpGqcmfgh9mgwi8Ux7CgS3RXxsjvLVrjY3i
-open_aea/agents/my_first_aea/aea-config.yaml,QmVr2EHMeAY1pXHPyh75JZUPKcY69P8QABnapjbZRh57p3
-open_aea/connections/scaffold,QmQeGqjcTvXn6sDGZCMUFTULjfHNW2RoCVN4PFeJUbbPWJ
-open_aea/connections/scaffold/connection.yaml,QmSNZgeHm1DRhhdzmsU3EKTTK4HyF4AU3dNEBaLFWF1XR7
-open_aea/contracts/scaffold,QmeqqfKY6pNLDbsCjBMxaNVdaHHd23NrpdHwYHhJ6DwqTB
-open_aea/contracts/scaffold/contract.yaml,QmZZ9g95RR9Yrbcp7ega7b4K8rFEs1r7Mp6Fr8qYwd5SZb
-open_aea/protocols/scaffold,QmSKEMPt7tC8pUukNnJzpMJrCTNxpgqVokgNBq1JToLB1t
-open_aea/protocols/scaffold/protocol.yaml,QmVJXv3czkzioSVmR3mqM58kdEEUbY9qc3xFw2V6PJbrrX
-open_aea/protocols/signing,QmYm7dnXSyWvKEWmPjUnt9MbY35Cro3CXDXP4Gh1shqKTf
-open_aea/protocols/signing/protocol.yaml,Qmef8wGZMCowEPbJ7GxfaNg6ga81ypD1L9Rw2dBkH9JqyH
-open_aea/skills/scaffold,QmYUhZb94UXmnb4B4S1Y4NbRKQBwYXextBiWTVRmMucrFv
-open_aea/skills/scaffold/skill.yaml,QmaH1Qo5MTCrkBZxtEr59XMW3gbN6PRdFoQXU1QcZSZ5QH
-valory/connections/p2p_libp2p,QmW1RV6HeSB1ZKYHNzoArmVcSHnsJHszKVdXWBkjrPQcBj
-valory/connections/p2p_libp2p/connection.yaml,QmciVjiK2XtYkwc5urQ8AniF6rCBXmHLMwZAjrqdBZgh5b
-valory/connections/p2p_libp2p_client,QmfYteGuDiyjCWjoB4EPfnubQYKtqnJzwtz2y3NAqXnX4B
-valory/connections/p2p_libp2p_client/connection.yaml,QmUkj83KxCcPFW5XFQfrE6WjWBU98RR5LgXiWRK4A9vc4r
-valory/connections/p2p_libp2p_mailbox,QmT5ordbbboo4tH7NoXBciXhU8ADUAddUQ516ytTBf2Fob
-valory/connections/p2p_libp2p_mailbox/connection.yaml,QmTUQVQgQFmV9iZRGT1m3oF8NdWyyQVex6VNCwKdLN9Ysu
-valory/protocols/acn,QmRXbv5sTYU9FeeDkfsWukhkuKW7L8d3cm83zG76VkEYdh
-valory/protocols/acn/protocol.yaml,QmZCeQcGyeuM3bWHBz4p2SfZDuNbiehRxcvzrqxhMzP1mo
-valory/protocols/tendermint,QmPm7CfYhEbfdAY77GGbvpAKEazUaXVMDNyPddcBhNYq5U
-valory/protocols/tendermint/protocol.yaml,QmNThVLHmqYjRv5rjckQYvT9xjJLFDmsK5ntDyGJMhtPjC
+fetchai/agents/error_test,QmZQycecx2LSRkxCJAhjD5MWhAcJq7cxq1Kh8BwXVc55m7
```

Comment:
From now we will use wrapper hashes for consistency (Since when we publish packages on IPFS node we use wrapper node hashes), and we don't need to store hashes for `.yaml` anymore we can just use these hashes to compare packages.

Reply:
Is that so? When using the `add` request against the node we used to get the hash on the LHS. Should we not use the same hash? Otherwise, users who end up using the vanilla IPFS node to produce their hashes end up with different hashes.

Reply:
Vanilla IPFS node also provides the wrapper node hash as the output for published file/directory.

Reply:
Ok, so you're saying we were always using the wrong hash (i.e. not the wrapper node hash)?